remote-access/ssh/passwordless.md

```diff
@@ -38,7 +38,7 @@ authorized_keys  id_rsa  id_rsa.pub  known_hosts
 
 The `id_rsa` file is your private key. Keep this on your computer.
 
-The `id_rsa.pub` file is your public key. This is what you share with machines you want to connect to. When the machine you try to connect to matches up your public and private key, it will allow you to connect.
+The `id_rsa.pub` file is your public key. This is what you share with machines that you connect to: in this case your Raspberry Pi. When the machine you try to connect to matches up your public and private key, it will allow you to connect.
```

Comment:
"you try" -> "you are trying" ?
"matches up" -> "successfully matches up" ?

Reply:
I don't think that first one is necessary - just adds more words.
Similarly, 'successfully' would be kind of redundant, and makes it more wordy.

It may be sensible to have a copy editor to look at this page anyway.

Reply:
We don't have the CE resource to do any serious (i.e. not just you two guys and me!) copy editing at the moment. I guess we need to start some sort of list of jobs that will need doing when CE resource becomes available, if ever. @aallan is going to be in charge of all documentation in the future, so I guess he is the best place for that.

Reply:
I see. Realistically, such a list would probably be something like "the whole documentation repository", since most of it was written without CE input.

A publicly accessible style guide would be useful - there was one back in the day, but I can't find it now.

I wonder if it would be possible to get hold of a copy editor to work on the documentation on a voluntary basis? I mean a professional copy editor, who is looking to volunteer their services.

Reply:
Actually, quite a few of the docs have been CE'd - our process before our CE person left to work full time for the Foundation meant all changes for probably 18 months went through CE. - so not only were changes checked, but the entire page in which they appeared as well. We do have an internal style guide, wonder if that could be repurposed specifically for the docs repo. 

I suspect getting a CE person to work for free won't be easy!

Reply:
I see. That's interesting. One possible avenue is goodmoves.org.uk - it's mostly for paid 3rd sector roles, but it also has volunteering opportunities for professionals on it. Or maybe if there is a national "3rd sector interface" that might be better. Although perhaps there's not enough for a volunteer copy editor to do, given someone, presumably a paid employee, would need to manage them.

Reply:
Definitely a style guide if you can provide a public version. I'm not sure anyone else would use it, but I'm sure there's stuff in there that would assist me, and perhaps save some of the back-and-forth we've had on some things, which takes up the time of paid employees.